composer.json

```diff
@@ -45,6 +45,8 @@
         "phpunit/phpunit": "^9.4",
         "sylius-labs/coding-standard": "^4.0",
         "symfony/console": "^4.4 || ^5.4",
+        "symfony/event-dispatcher": "^4.4 || ^5.4",
```

Comment:
The event dispatcher is used in a few classes, but even for development, it was only installed through transient dependencies.